server/build.gradle

```diff
@@ -26,19 +32,21 @@ mainClassName = 'umm3601.Server'
 // External dependencies that our application utilizes
 dependencies {
   // Google core libraries for Java, various useful utilities
-  implementation 'com.google.guava:guava:28.2-jre'
+  implementation 'com.google.guava:guava:30.1-jre'
 
   // Javalin, a simple web framework for Java
-  implementation 'io.javalin:javalin:3.7.0'
+  implementation 'io.javalin:javalin:3.12.0'
 
   // Mongo DB Driver for Java
-  implementation 'org.mongodb:mongodb-driver:3.12.5'
+  implementation 'org.mongodb:mongodb-driver:3.12.7'
 
   // Jackson, a JSON library for Java
   implementation 'com.fasterxml.jackson.core:jackson-databind:2.10.2'
+  // implementation 'com.fasterxml.jackson.core:jackson-databind:2.12.0'
```

Comment:
This and the next line are my failed attempts to upgrade the Jackson libraries. If we accept this PR I'll probably delete those comments before merging in, and instead create new GitHub issues about these upgrades.